src/app/package/page.tsx

```diff
@@ -24,14 +27,24 @@ export default function PackageList() {
     useGetPackageDetailQuery(PACKAGE['premium'])
   const { data: latest_subscription = {} as LatestSubscriptionResponse } =
     useGetLatestSubscriptionQuery()
+  const [createTransaction, { data, isLoading }] =
+    useCreateTransactionMutation()
 
+  const handleCreateTransaction = async () => {
+    createTransaction({ package_id: premium_package.id })
+  }
   const features: (keyof PackageDetailResponse)[] = [
     'event_planner',
     'event_tracker',
     'event_timeline',
     'event_rundown',
     'ai_assistant',
   ]
+  useEffect(() => {
+    if (!!data) {
```

Comment:
Based on analysis on sonarcloud, this type of redundant double negation will cause code smell on our codes. I get that you want to cast it to boolean, but perhaps to minimize the amount of code smells, it's better to not use double negation.

The reason for this could be seen here:
https://sonarcloud.io/organizations/reveliostartup/rules?open=typescript%3AS6509&rule_key=typescript%3AS6509

Reply:
okay, thanks for the info